Makefile

```diff
@@ -446,13 +446,23 @@ ifeq ($(CI), true)
 	gocov convert $(REPORTS)/$(TEST_SCENARIO)_coverage.out | gocov-xml > $(REPORTS)/$(TEST_SCENARIO)_coverage.xml
 endif
 
-unit-test:
+docker-db:
```

Comment:
Can we call this `run-db-container`?

This will match the `kill-db-container` name.